api/v1beta2/mysqlcluster_types.go

```diff
@@ -117,6 +117,11 @@ type MySQLClusterSpec struct {
 	// If set to true, the sidecar container is not added. The default is false.
 	// +optional
 	DisableSlowQueryLogContainer bool `json:"disableSlowQueryLogContainer,omitempty"`
+
+	// MySQLDLocalHost configures mysqld interface to bind and be accessed over localhost instead of pod name.
+	// During container init moco-agent will set mysql admin interface is bound to localhost. The moco-agent will also
+	// communicate with mysqld over localhost when acting as a sidecar.
+	MySQLDLocalHost bool `json:"mysqldLocalHost,omitempty"`
```

Comment:
I apologize for the delayed response.

I find the option name `mysqldlocalHost` a bit unclear.
Perhaps something like `useLocalhost` or `agentUseLocalhost` might be clearer?
Of course, if there is a better suggestion, I'm open to that as well.

Since changes to the CRD are not trivial, I want to proceed with caution.

Reply:
No problem, will update it to `agentUseLocalhost`